lib/config/defaults.js

```diff
@@ -984,7 +981,7 @@ const applyOutputDefaults = (
 					if (tp.nodeBuiltins) return "async-node";
 					break;
 				case "module":
-					if (tp.dynamicImport) return "import";
+					if (tp.dynamicImport || output.module) return "import";
```

Comment:
I improved it. Why?

- We don't know target and enviroment 
- But we set `chunkFormat: "module"` only when `output.module` is true and `tp.dynamicImport` is true
- We can load `chunkFormat: "module"` only using `chunkLoading: "import"`
- So let's use `chunkLoading: "import"` when `tp.dynamicImport` or `output.module` is true

So when we can't undestand is `dynamicImport` supported or not and you ask `module` chunk format, we suppose your env supported `import(...)`

Reply:
Otherwise it doesn't make sense, you can't load `chunkFormat: "module"` using other methods

Reply:
Yeah，i think this is indeed more reasonable.